cypress/support/commands.js

```diff
@@ -1,7 +1,9 @@
 Cypress.Commands.add('onLoad', () => {
     const baseDataURL = 'https://pokeapi.co/api/v2/pokemon';
 
-    cy.intercept(`${ baseDataURL }?limit=151`, {
+    // got an error because our map changes the url to be able to access the image, see how can we add the image url as well 
+
+    cy.intercept(`${ baseDataURL }?limit=6`, {
         pokeDex:[
             {
```

Comment:
Great job with the command. I like that we can stub all of the data for every test before hand and really gain maximum control of our test environment and expected outcomes.